src/AcceptanceTests/ConfigureEndpointSqsTransport.cs

```diff
@@ -27,7 +27,7 @@ public Task Configure(string endpointName, EndpointConfiguration configuration,
                 }
             }
 
-            settings.TestExecutionTimeout = TimeSpan.FromSeconds(20);
+            settings.TestExecutionTimeout = TimeSpan.FromSeconds(20 * 2);
```

Comment:
Why doing math instead of 40 seconds?

How about setting it to 40 seconds and then write a comment that the value has to be greater than 30 secs (visibility timeout)?

Reply:
I have no preference, up to @SeanFeldman to decide 😄